src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -262,10 +262,12 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
       this.currentStep = E2EIHandlerStep.INITIALIZED;
     }
 
+    const isFreshMLSSelfClient = void this.coreE2EIService.isFreshMLSSelfClient();
```

Comment:
This is very weird, voiding the result of the function will always yield `undefined`